test-requirements.txt

```diff
@@ -6,6 +6,7 @@ hacking<0.11,>=0.10.0
 
 coverage>=3.6
 python-subunit>=0.0.18
+retry>=0.9.2
```

Comment:
Seems like retrying != retry. I will install retrying and get the tests to pass accordingly.